instrumentation/apache-dubbo-2.7/testing/src/main/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/DubboTestUtil.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.apachedubbo.v2_7;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import org.apache.dubbo.config.bootstrap.DubboBootstrap;
+
+/** compatible with dubbo3.x and dubbo 2.7 */
+class DubboTestUtil {
+
+  private DubboTestUtil() {}
+
+  static Object newFrameworkModel() {
+    try {
+      // only present in latest dep
+      return Class.forName("org.apache.dubbo.rpc.model.FrameworkModel")
+          .getDeclaredConstructor()
+          .newInstance();
+    } catch (ReflectiveOperationException exception) {
+      return null;
+    }
+  }
+
+  @SuppressWarnings("ThrowSpecificExceptions")
```

Comment:
You could have just thrown `IllegalStateException` to avoid needing this suppression